x-pack/examples/triggers_actions_ui_example/server/connector_types/system_log_example.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { schema, TypeOf } from '@kbn/config-schema';
+
+import { LogMeta } from '@kbn/core/server';
+import type {
+  ActionType as ConnectorType,
+  ActionTypeExecutorOptions as ConnectorTypeExecutorOptions,
+  ActionTypeExecutorResult as ConnectorTypeExecutorResult,
+} from '@kbn/actions-plugin/server/types';
+import {
+  AlertingConnectorFeatureId,
+  UptimeConnectorFeatureId,
+} from '@kbn/actions-plugin/common/connector_feature_config';
+import { ConnectorAdapter } from '@kbn/alerting-plugin/server';
+
+// see: https://en.wikipedia.org/wiki/Unicode_control_characters
+// but don't include tabs (0x09), they're fine
+const CONTROL_CHAR_PATTERN = /[\x00-\x08]|[\x0A-\x1F]|[\x7F-\x9F]|[\u2028-\u2029]/g;
+
+// replaces control characters in string with ;, but leaves tabs
+function withoutControlCharacters(s: string): string {
+  return s.replace(CONTROL_CHAR_PATTERN, ';');
+}
+
+export type ServerLogConnectorType = ConnectorType<{}, {}, ActionParamsType>;
+export type ServerLogConnectorTypeExecutorOptions = ConnectorTypeExecutorOptions<
+  {},
+  {},
+  ActionParamsType
+>;
+
+// params definition
+
+export type ActionParamsType = TypeOf<typeof ParamsSchema>;
+
+const ParamsSchema = schema.object({
+  message: schema.string(),
+});
+
+export const ConnectorTypeId = '.system-log-example';
+// connector type definition
+export function getConnectorType(): ServerLogConnectorType {
+  return {
+    id: ConnectorTypeId,
+    isSystemActionType: true,
+    minimumLicenseRequired: 'gold', // Third party action types require at least gold
```

Comment:
Probably not. I will change it to `basic`.

Reply:
When registering the action type the `ensureSufficientLicense` in `x-pack/plugins/actions/server/lib/ensure_sufficient_license.ts` will be thrown an error because only the `.server-log` and the `.index` are allowed to be on the basic license. What about leaving it as it is from now on and checking with @shanisagiv1 the minimum required license for system actions?

Reply:
Sounds good.